src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -2033,42 +2046,67 @@ void CmdSketcherConstrainCoincident::activated(int iMsg)
 
 void CmdSketcherConstrainCoincident::applyConstraint(std::vector<SelIdPair> &selSeq, int seqIndex)
 {
+    SketcherGui::ViewProviderSketch* sketchgui = static_cast<SketcherGui::ViewProviderSketch*>(getActiveGuiDocument()->getInEdit());
```

Comment:
Just a stylistic comment, you don't have to change it here, but in future code: when using `static_cast`, `dynamic_cast`, etc. where we already explicitly state the type, that's a good place to use `auto` for the variable definition. Sort of a micro-example of the DRY principle.

Reply:
This code has just been moved.
But I take note of your message. I will try to use auto in those situation in the future. 
If I come accross such occurence while making a PR, should I always fix such things along?

Reply:
It's hard to make a general statement about it: knowing now that this code was simply relocated, then no, I'd say leave it alone. Switching to `auto` is a minuscule change that makes almost no practical difference, so the very small increase in readability is not worth complicating the review. On the other hand, if you ran across an old-style for loop, and you were already touching that line of code for some other reason, then I'd say yes, let's go ahead and switch to a ranged-for. Brand new code is different, but working with old code IMO there's more wiggle room. There isn't a "policy" either way, we just discuss it in the PR review process and come to a consensus. You are always welcome to push back and tell me why you don't think a particular suggestions makes sense in this particular context: I won't take it personally, we're both trying to make incremental improvements to the code.

Reply:
It's hard to make a general statement about it: knowing now that this code was simply relocated, then no, I'd say leave it alone. Switching to `auto` is a minuscule change that makes almost no practical difference, so the very small increase in readability is not worth complicating the review. On the other hand, if you ran across an old-style for loop, and you were already touching that line of code for some other reason, then I'd say yes, let's go ahead and switch to a ranged-for. Brand new code is different, but working with old code IMO there's more wiggle room. There isn't a "policy" either way, we just discuss it in the PR review process and come to a consensus. You are always welcome to push back and tell me why you don't think a particular suggestions makes sense in this particular context: I won't take it personally, we're both trying to make incremental improvements to the code.